Framework/PythonInterface/plugins/functions/PrimStretchedExpFT.py

```diff
@@ -35,8 +35,8 @@
 
 class PrimStretchedExpFT(IFunction1D):
     # pylint: disable=super-on-old-class
-    def __init__(self):
-        super(self.__class__, self).__init__()
+    def __init__(self, _is_called_by_function_factory):
```

Comment:
Is there anyway to shield users from this? It's going to be a tough one to explain what that argument means.

Would a fallback option be to use something like
```python
def __init__(self, *args):
    super(self.__class__, self).__init__(*args)
```

to be less explicit in the future if we need to provide flexibility for other arguments?